scss/_nav.scss

```diff
@@ -76,13 +82,6 @@
   .nav-link {
     @include border-radius($nav-pills-border-radius);
   }
-
```

Comment:
Are you talking about this one?
```scss
&.active {
  color: $nav-pills-active-link-color;
  cursor: default;
  background-color: $nav-pills-active-link-bg;
}
```
Or that one?
```scss
.nav-link {
  @include border-radius($nav-pills-border-radius);
}
```
For info the second one wasn't modified in this PR.

Anyway in both cases I can't reproduce the problem you are reporting regarding navs with dropdown.
Could you clarify in which way it breaks it ? Or point me to an exemple in the docs that is broken? Or provide more details?